Tests/OktaOidcTests/OIDExternalUserAgentNoSsoTests.swift

```diff
@@ -23,7 +23,7 @@ class OKTExternalUserAgentRequestMock: OKTExternalUserAgentRequest {
     }
     
     func redirectScheme() -> String! {
-        return "com.okta.callback://oauth/callback"
+      NSURL(string: "com.okta.callback://oauth.callback")!.scheme
```

Comment:
Why not just return `com.okta.callback` instead?

Reply:
😄  
You're right. I fixed that.